cmake/ctest/drivers/geminga/ctest_linux_nightly_mpi_release_muelu_kokkos_refactor_cuda_geminga.cmake

```diff
@@ -75,29 +75,30 @@ SET(Trilinos_PACKAGES MueLu Xpetra Amesos2)
 
 SET(EXTRA_CONFIGURE_OPTIONS
   ### ETI ###
-  "-DTrilinos_ENABLE_EXPLICIT_INSTANTIATION:BOOL=ON"
-    "-DTpetra_INST_INT_LONG_LONG:BOOL=OFF"
-    "-DTrilinos_ENABLE_COMPLEX:BOOL=ON"
-    "-DTeuchos_ENABLE_COMPLEX:BOOL=OFF"
-    "-DTpetra_INST_COMPLEX_DOUBLE:BOOL=OFF"
-    "-DTpetra_INST_COMPLEX_FLOAT:BOOL=OFF"
-    "-DTpetra_INST_SERIAL=ON"
+  "-D Trilinos_ENABLE_EXPLICIT_INSTANTIATION:BOOL=ON"
+    "-D Tpetra_INST_INT_LONG_LONG:BOOL=OFF"
+    "-D Tpetra_INST_INT_INT:BOOL=ON"
+    "-D Trilinos_ENABLE_COMPLEX:BOOL=OFF"
+    "-D Teuchos_ENABLE_COMPLEX:BOOL=OFF"
```

Comment:
You don't need to set `Teuchos_ENABLE_COMPLEX`; the Trilinos flag should work.  Does it not?

Reply:
Yes that's right, I can also remove the other two lines:
```
    "-D Tpetra_INST_COMPLEX_DOUBLE:BOOL=OFF"
    "-D Tpetra_INST_COMPLEX_FLOAT:BOOL=OFF"
```
for the same reason.

Reply:
@lucbv yes please :-) There are also separate options `Trilinos_ENABLE_COMPLEX_DOUBLE` and `Trilinos_ENABLE_COMPLEX_FLOAT`, if you just want one but not the other.